### DISCUSSION TITLE
Dreame D10plus gen 2

### DISCUSSION THREAD

**Discussion by Broomstick1978** [Tasshack/dreame-vacuum#717] [Q&A] [answered]
Dear developer, I have bougt a new Dreame vacuum cleaner robot. After checking, I found that my model is dreame_vacuum_r2423_miapC82E, which is not in the compatible list unfortunately.
This product is an upgraded version of D10plus, released on september 3, 2024. 
Can u please update the list with this vacuum.

Thank you for your hard work and dedication in developing this open source project.

With regard,
Erik den Breems

**Comment by Tasshack** [comment, marked as answer]
https://github.com/Tasshack/dreame-vacuum/blob/dev/docs/supported_devices.md
